DuckDuckGo/Application/AppDelegate.swift

```diff
@@ -233,7 +235,7 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
                 appearancePreferences: .shared,
                 pinnedTabsManager: pinnedTabsManager,
                 internalUserDecider: internalUserDecider,
-                configurationStore: ConfigurationStore.shared,
+                configurationStore: ConfigurationStore(),
```

Comment:
Similar to iOS, we may wish to set up a single instance of this in a similar place that we do with other dependencies, and pass it around - I don't trust that the codebase isn't set up with a single instance in mind in places we're not aware of.